docs/using-tre/tre-for-research/importing-exporting-data-airlock.md

```diff
@@ -82,17 +82,26 @@ Exporting data from a secure TRE workspace to the outside world involves similar
 
```

Comment:
In the note regarding the Airlock Notifier, the sentence `Please check with your TRE administrator` is unclear. It would be better to remove it or clarify what the user should check with their administrator.